src/git-duet/git_config.go

```diff
@@ -114,6 +122,33 @@ func (gc *GitConfig) RotateAuthor() (err error) {
 	return nil
 }
 
+func makeRotatedTeam(teamAuthor *Pair, teamCommitter *Pair) (newAuthor *Pair, newCommitter *Pair) {
```

Comment:
This function is difficult to read, and has a lot of unnecessary code.  The name also doesn't really make sense.  It can be simplified, for example:

```go
func rotatePair(teamAuthor *Pair, teamCommitter *Pair) (newAuthor *Pair, newCommitter *Pair) {
    initialsList := append(strings.Split(teamCommitter.Initials, ", "), teamAuthor.Initials)
    nameList := append(strings.Split(teamCommitter.Name, ", "), teamAuthor.Name)
    emailList := append(strings.Split(teamCommitter.Email, ", "), teamAuthor.Email)
    newAuthor = &Pair{
        Name:     nameList[0],
        Email:    emailList[0],
        Initials: initialsList[0],
    }
    newCommitter = &Pair{
        Name:     strings.Join(nameList[1:], ", "),
        Email:    strings.Join(emailList[1:], ", "),
        Initials: strings.Join(initialsList[1:], ", "),
    }
    return newAuthor, newCommitter
}
```

Incidentally, this will work with any number of committers including exactly one, so you can remove the `strings.Contains(committer.Initials, ",")` check above and simplify that part of the method to:

```go
newAuthor, newCommitter := rotatePair(author, committer)
```